pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -252,22 +267,25 @@ pub mod pallet {
 					denomination,
 				)?;
 
-				// TODO: reset team account
+				T::Fungibles::reset_team(
+					asset_id.clone(),
+					pool_id.clone().into(),
+					entry.team.admin.clone(),
+					entry.team.issuer.clone(),
+					entry.team.freezer.clone(),
+				)?;
+
+				if !entry.tradable {
+					T::Fungibles::freeze_asset(pool_id.clone().into(), asset_id.clone())?;
```

Comment:
unfortunately, this will lead to all mint/burn operations failing, as these all happen after an 'asset-is-live' check in the assets pallet. You will have to unfreeze the asset before any change to balances in the pallet. An alternative is placing locks on any amount minted I guess.